client/lib/abtest/index.js

```diff
@@ -55,6 +55,16 @@ export const getABTestVariation = name => new ABTest( name ).getVariation();
  */
 export const getSavedVariations = () => store.get( ABTEST_LOCALSTORAGE_KEY ) || {};
 
+/**
+ * Set the variation for a test - useful for testing!
+ *
+ * @param {String} name - The name of the A/B test
+ * @param {String} variation - The test variation to set
+ * @returns {undefined}
+ */
+export const setABTestVariation = ( name, variation ) =>
```

Comment:
Nit: `save` instead of `set`?
```suggestion
export const saveABTestVariation = ( name, variation ) =>
```